IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CommunicationsApplianceTypeTelecommand/DocPropertySet.xml

```diff
@@ -4,7 +4,7 @@
 		<DocProperty xsi:nil="true" href="NumberOfDiagnosticMaintenanceWorkstations_1NteTJwCTDPxZ71U8R760" />
 		<DocProperty xsi:nil="true" href="NumberOfCPUs_1rjaY2H_LBUB1VPn5lebft" />
 		<DocProperty xsi:nil="true" href="NumberOfTrafficElectricalTractionWorkstations_2lLmk3LIf6ew_EjtIAMdjz" />
-		<DocProperty xsi:nil="true" href="HasBeenDiagnosed_1s3CI0yTT8hAw1ZrWl4DYo" />
+		<DocProperty xsi:nil="true" href="IsDiagnosed_1s3CI0yTT8hAw1ZrWl4DYo" />
```

Comment:
I  assume this is for O&M, but this i feel is far to simplistic how do you know when it had the diagnostic? surely that is the more important. I would apply some sort of task or use a work order. If it is needed as part of hand over and such then ok.

Reply:
@fhulin